src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
it's better to use more significant names of variables to make code readable - try to cheange all similar places in file
```suggestion
    ? people.filter((person) => person.sex === 'f'
      && people.find((child) => child.mother === person.name))
```